projects/aca-shared/src/lib/services/app.service.ts

```diff
@@ -116,6 +119,10 @@ export class AppService implements OnDestroy {
     );
   }
 
+  getSidenavValue(str: boolean) {
```

Comment:
this is called "get value" but it is not getting anything, just emitting "next"... why parameter is called "str" and and has Boolean type? can you explain the meaning of all this here, thanks